src/components/organisms/Header/Header.tsx

```diff
@@ -207,10 +205,13 @@ const ItemLink = ({
         );
     };
     return (
-        <Link href={link} className='h-full' passHref>
-            <a className='h-full' href='_' data-cy={dataCy.toLowerCase()}>
-                <NavTab text={text} active={useCheckActive()} />
-            </a>
+        <Link
+            href={link}
+            className='h-full'
+            passHref
```

Comment:
@as472 I've removed it from `src/components/pages/Landing/Landing.tsx` because the child of the <Link> is a string

But I've left it in Header because NavTab is passed as a child.